README.md

```diff
@@ -63,6 +64,8 @@ podman push quay.io/my-container-registry-namespace/my-manifest-bundle:latest
 
 Of course, this build step can be done with any other OCI spec container tools like `docker`, `buildah`, `libpod`, etc.
 
+Note that you do not need to create your bundle manually. [Operator-SDK](https://github.com/operator-framework/operator-sdk) provide features and helpers to build, to update, to validate and to test bundles for projects which follows the SDK layout or not. For more information check its documentations over [Integration with OLM](https://sdk.operatorframework.io/docs/olm-integration)
```

Comment:
@dinhxuanvu

is it making more sense now?

Reply:
That does sound better.